Legion.FrontEnd/src/events/EventsCalendar/Components/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import moment from 'moment';
+import ChevronLeftIcon from 'mdi-react/ChevronLeftIcon';
+import ChevronRightIcon from 'mdi-react/ChevronRightIcon';
+
+class Header extends React.PureComponent {
+  state = {
+    month: this.props.current,
+  }
+  
+  previousMonth = (month) => {
+    this.setState({ month: month.subtract(1, 'M') })
+  }
+  
+  nextMonth = (month) => {
+    this.setState({ month: month.add(1, 'M') })
+  }
+
+  render() {
+    const month = this.state.month
+    return(
+      <div className='header row flex-middle'>
+        <div><ChevronLeftIcon className='column column-end' onClick={this.previousMonth(month)} /></div>
```

Comment:
`onClick()` event not functioning

Reply:
Ah, that's because this needs to be a higher-order function.
```jsx
previousMonth = month => () => {
  // code
}
```